app/app.cpp

```diff
@@ -1,12 +1,58 @@
 #include "app.h"
 
 #include <format>
+#include <cmath>
+#include <complex>
+#include <numeric>
 
 namespace vsite::oop::p1 {
 
-/*
-* function implementations
-*/
+	/*
+	* function implementations
+	*/
+	double f1(const int x, const int y)
+	{
+		return std::pow(static_cast<int>(x), static_cast<double>(y));
+	}
 
+	bool f2(const bool expression)
+	{
+		return not expression;
```

Comment:
logički _not_ je `!` (i u C-u i u C++-u)

Reply:
Jasno, to sam i prvobitno bio napisao, ali sam isao provjeriti postoji li not operator u c++ i postoji, pa sam odlucio tako napisati. -> [cppreference link](https://en.cppreference.com/w/cpp/keyword/not)